src/main/java/ch/uzh/ifi/hase/soprafs21/service/ActivityService.java

```diff
@@ -150,19 +145,19 @@ public void setSwipingStatus(long activityId, String token, SwipeStatus swipeSta
     }
 
     public Set<Activity> getAllUnmatchedActivities(User user){
-        Set<Activity> allUnmatchedActivities = new HashSet<>();
+        Set<Activity> allUnmatchedActivites = new HashSet<>();
```

Comment:
please do not rename / btw _activities_ should be correct?
=> Update everywhere